wp-content/plugins/core/src/Blocks/Types/Hero.php

```diff
@@ -9,39 +9,35 @@
 use Tribe\Gutenpanels\Blocks\Sections\Toolbar_Section;
 use Tribe\Project\Blocks\Block_Type_Config;
 
-/*
- * @TODO: THIS IS NOT COMPLETE AND WILL BE PICKED BACK UP BY RYAN
- */
-
 class Hero extends Block_Type_Config {
 
 	public const NAME = 'tribe/hero';
 
-	public const BACKGROUND_IMAGE = 'bg-image';
-	public const LEAD_IN          = 'lead-in';
-	public const TITLE            = 'title';
-	public const DESCRIPTION      = 'description';
-	public const CTA              = 'cta';
-	public const LAYOUT           = 'layout';
-	public const LAYOUT_LEFT      = 'alignleft';
-	public const LAYOUT_CENTER    = 'aligncenter';
+	public const IMAGE         = 'image';
```

Comment:
Changes in this file are to create consistency with the existing Interstitial panel. Additionally, Jonathan and I will be reviewing how to align the block registration with the FE templates later today. I'll be opening separate PRs for each panel with that work later.